site/src/utils/utils.js

```diff
@@ -8,17 +8,20 @@ export async function notFound(event) {
 
 /**
  * @template T
- * @param {() => Promise<T | void>} fn
+ * @param {(ctx : import("../bindings").RouteContext) => Promise<T | void>} fn
  * @param {string} label
+ * @param {import("../bindings").RouteContext} ctx
  * @returns {Promise<T | void>}
  */
-export async function timed(fn, label) {
-  console.log(`START: ${label}`)
-  console.time(`END: ${label}`)
+export async function timed(fn, label, ctx) {
```

Comment:
This is no longer timing anything. Is it possible to log the time spent with sentry?

Reply:
there no timing inside workers, date.now console timing etc dont work